src/chainer.js

```diff
@@ -8,11 +10,12 @@ function chainer(functions) {
     let result = x;
 
     for (const f of functions) {
-      result = f(result);
+      if (typeof f === 'function') {
+        result = f(result);
+      }
     }
-
     return result;
```

Comment:
There is some inconsistency in function declaration. Sometimes you are using function expression (with semicolon at the end), sometimes you are using function declaration (without semicolon). Stick to one style for consistency.